docs/using-connector.md

```diff
@@ -77,6 +78,17 @@ conf.set("spark.riak.connections.max", "50")
 sc = pyspark.SparkContext("spark://127.0.0.1:7077", "test", conf)
 ```
 
+
+## Failover Handling
+ 
+When reading data from Riak, the Spark-Riak connector will run a full bucket query. This query first obtains metadata, called a coverage plan, from Riak about 
+where the data is located in the cluster. This coverage plan will have a list of nodes were the data lives, but the Spark-Riak connector will use the first entry 
```

Comment:
Actually connector will use all entries for reading all data. The text related to "first entry" should be removed